config/routes.rb

```diff
@@ -1,11 +1,17 @@
 Rails.application.routes.draw do
+  root 'student#index'
+
+  get 'student/new'
```

Comment:
These routes seem wrong to me. The way `get` works is you say what url you're looking for, then, when the system gets that url, where should it go? You haven't told it where to go. But that all is taken care of for you if you use:

```ruby
resources :students
```

This makes all the routes you need for index, show, new, create, edit, update, and delete. I'd highly recommend using that instead. Then you can get rid of all these get's in here.